src/main/java/com/github/caldav4j/util/MethodUtil.java

```diff
@@ -26,13 +25,6 @@
 	public static int StatusToExceptions(HttpRequestBase method, HttpResponse response) throws CalDAV4JException {
 		if (method != null && response != null) {
 			int status = response.getStatusLine().getStatusCode();
-			if (log.isDebugEnabled()) {
```

Comment:
Why remove this part?

Reply:
Usage of `EntityUtils.toString(response.getEntity()` is causing an exception in a UnitTest (PutGetTest#testAddRemoveCalendarResource) when the log is enabled for DEBUG due to closed content stream of response entity and the only reason to check if the debug is enabled it to print the entity's content.